packs/feat-effects/effect-crystal-luminescence.json

```
@@ -0,0 +1,49 @@
+{
+    "_id": "1bzFIIEw2K1CAUe9",
+    "img": "systems/pf2e/icons/spells/prismatic-wall.webp",
+    "name": "Effect: Crystal Luminescence",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Crystal Luminescence]</p>\n<p>If a spell or ability would activate your glowing horn while Crystal Luminescence is active, it instead increases the radius of the bright light and dim light by 10 feet each until the start of your next turn. This isn't cumulative, so using another such ability doesn't increase the radius again.</p>"
```

```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Crystal Luminescence]</p>\n<p>The radius of the bright light and dim light of your crystal luminescence increase by 10 feet.</p>"
```
We can leave the full rules text to the feat itself to make effects more clear in what they actually automate